pages/_app.js

```diff
@@ -1,10 +1,25 @@
 import GlobalStyle from "../styles";
+import useSWR from "swr";
+import { SWRConfig } from "swr";
+import ArtPieces from "@/components/ArtPieces";
+import Navigation from "@/components/Navigation";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
 
 export default function App({ Component, pageProps }) {
+
+  const { data, error,  isLoading } = useSWR('https://example-apis.vercel.app/api/art', fetcher)
+  const pieces = data
+  console.log("Pieces:", pieces)
+
   return (
     <>
       <GlobalStyle />
-      <Component {...pageProps} />
+      <SWRConfig value={{ fetcher }}>
```

Comment:
I think we don't need the `SWRConfig`  and set an application-wide fetcher function. We will fetch data at the beginning only and pass to children with `useSWR` hook, which you already implemented. You can remove this.